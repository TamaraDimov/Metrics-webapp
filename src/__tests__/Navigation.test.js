import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import store from '../redux/store';
import Navbar from '../components/Navigation';

describe('Navbar', () => {
  it('renders component correctly', () => {
    const { container } = render(
      <Provider store={store}>
        <BrowserRouter>
          <Navbar />
        </BrowserRouter>
      </Provider>,
    );
    expect(container).toMatchSnapshot();
  });
});
