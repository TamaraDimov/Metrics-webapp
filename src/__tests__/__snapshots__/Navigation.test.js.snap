// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar testing render component 1`] = `
<div
  className="navbar"
>
  <a
    aria-current="page"
    className="link active"
    href="/"
    onClick={[Function]}
  />
</div>
`;
